src/components/user/SignUpForm.js

```diff
@@ -10,15 +11,20 @@ import {
 
 import Loading from 'components/common/Loading';
 import Input from 'components/common/Input';
+import Select from 'components/common/Select';
 import { validations, signUp } from 'utils/constraints';
 
 const messages = defineMessages({
   name: { id: 'login.form.name' },
   email: { id: 'login.form.email' },
   password: { id: 'login.form.password' },
-  passConfirmation: { id: 'signup.form.passconfirmation' }
+  passwordPlaceHolder: { id: 'login.form.password.placeholder' },
+  passConfirmation: { id: 'signup.form.passconfirmation' },
+  gender: { id: 'signup.form.gender' }
 });
 
+const options = [{ value: 'female', label: 'Female' }, {value: 'male', label: 'Male'}, {value:'other', label: 'Other'}];
```

Comment:
Move this to a constant file

Reply:
what about the internationalization ?  What should I import?